tests/legacy-cli/e2e/initialize/500-create-project.ts

```diff
@@ -20,7 +20,7 @@ export default async function () {
     // Ensure local test registry is used when outside a project
     await setNPMConfigRegistry(true);
 
-    await ng('new', 'test-project', '--skip-install');
+    await ng('new', 'test-project', '--skip-install', '--no-zoneless');
```

Comment:
In a followup we should likely try to have most of the our tests test the default behavior (IE: zoneless).